### PR TITLE
1443 automated bulk file processing 

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Checklist
Reminder: If any change made to this repo affects the acceptance tests then the version pin in the Pipfile in [census-rm-acceptance-tests]() needs to be updated.
* [ ] Updated census-rm-acceptance-tests version pin (if applicable)

# Motivation and Context

We're gonna automate the bulk processors to save time during Census. While doing this we've updated the logging to json formatting so now it can be picked up by our GCP metrics.

# What has changed

- Added script to automate bulk processors
- Updated logging

# How to test?

- Build a toolbox image from the PR and push it to your local GCR.
- Deploy the new cronjob bulk processor in the kubernetes PR
- The cronjob should run the automate_bulk_processor script in toolbox on the hour every hour
- Put a couple of files in the bulk buckets and the script should process them all successfully
- Throw in some bad messages and you can see the failures be picked up metrics
- Make the refusal bulk processor fail and it'll show some error lines in the script but carry on the through the rest of the bulk processor.

# Links

[Trello](https://trello.com/c/eXayJFQQ/)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[Automated bulk file processing (5)](https://trello.com/c/eXayJFQQ/1443-automated-bulk-file-processing-5)

**Review by AdamHawtin — APPROVED** [review]
LGTM

**Comment by AdamHawtin** [comment]
Worth noting the AT's need the toolbox ref pin updating too

File: automating_bulk_processor.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ret_value=$?
+if [ $ret_value -ne 0 ];
+then
+  echo "address_update_processor has failed"
+fi
+python -m toolbox.bulk_processing.uninvalidate_address_processor
+ret_value=$?
+if [ $ret_value -ne 0 ];then
+    echo "uninvalidate_address_processor has failed"
+fi
```
Comment (NickGrantX) [inline automating_bulk_processor.sh:49]: the script can/should return a non-zero exit code so the k8s job indicates an error

File: automating_bulk_processor.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+python -m toolbox.bulk_processing.refusal_processor
+ret_value=$?
+if [ $ret_value -ne 0 ];
+then
+  echo "refusal_processor has failed"
```
Comment (NickGrantX) [inline automating_bulk_processor.sh:9]: these echo'ed errors will be harder to grep out of the logs won't they?

**Review by NickGrantX — APPROVED** [review]
Happy to approve - good work - but this is missing a couple of suggestions I made on the spike

File: automating_bulk_processor.sh (hunk 2 above)
Comment (ryangrundy7) [reply automating_bulk_processor.sh:9]: I've created a metric in the terraform that picks up the log lines and we can alert on that

File: automating_bulk_processor.sh (hunk 1 above)
Comment (ryangrundy7) [reply automating_bulk_processor.sh:49]: I've just updated the script now to error on failure so give it a go and see how it runs. I had to change the k8s cronjob yaml to have a backoff of 0 to allow it to fail correctly

**Review by NickGrantX — APPROVED** [review]
LGTM

**Pull request merged**